src/til/ut_til/RectangleTests.cpp

```diff
@@ -300,22 +300,26 @@ class RectangleTests
 
     TEST_METHOD(Boolean)
     {
-        BEGIN_TEST_METHOD_PROPERTIES()
-            TEST_METHOD_PROPERTY(L"Data:left", L"{0,10}")
-            TEST_METHOD_PROPERTY(L"Data:top", L"{0,10}")
-            TEST_METHOD_PROPERTY(L"Data:right", L"{0,10}")
-            TEST_METHOD_PROPERTY(L"Data:bottom", L"{0,10}")
-        END_TEST_METHOD_PROPERTIES()
+        SetVerifyOutput verifyOutputScope{ VerifyOutputSettings::LogOnlyFailures };
 
-        til::CoordType left, top, right, bottom;
-        VERIFY_SUCCEEDED_RETURN(TestData::TryGetValue(L"left", left));
-        VERIFY_SUCCEEDED_RETURN(TestData::TryGetValue(L"top", top));
-        VERIFY_SUCCEEDED_RETURN(TestData::TryGetValue(L"right", right));
-        VERIFY_SUCCEEDED_RETURN(TestData::TryGetValue(L"bottom", bottom));
+        static constexpr til::CoordType values[] = { til::CoordTypeMin, -1, 0, 1, til::CoordTypeMax };
 
-        const auto expected = left < right && top < bottom;
-        const til::rect actual{ left, top, right, bottom };
-        VERIFY_ARE_EQUAL(expected, (bool)actual);
+        for (const auto left : values)
+        {
+            for (const auto top : values)
+            {
+                for (const auto right : values)
+                {
+                    for (const auto bottom : values)
+                    {
+                        const til::rect r{ left, top, right, bottom };
+                        const auto expected = left >= 0 && top >= 0 && right > left && bottom > top;
+                        const auto actual = static_cast<bool>(r);
+                        VERIFY_ARE_EQUAL(expected, actual);

```

Comment:
Make sure you log which case this is failing in, otherwise you just get a bunch of `passed passed passed passed failed passed failed passed`

Reply:
I did this because it otherwise emits at least 500+ lines of passed text. I think since a failure is unlikely, this is fine.